client/van_site_config_create.go

```diff
@@ -18,7 +18,18 @@ func (cli *VanClient) VanSiteConfigCreate(ctx context.Context, spec types.VanSit
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "skupper-site",
 		},
-		Data: map[string]string{},
+		Data: map[string]string{
+			"name":                   types.DefaultVanName,
+			"edge":                   "false",
+			"service-controller":     "true",
+			"service-sync":           "true",
+			"console":                "false",
+			"router-console":         "false",
+			"console-authentication": "",
```

Comment:
console-authentication could default to 'internal' (that might not be a great name, but it is the current default behaviour, so would be consistent with the aim of this patch to show that in the config I think). (Again, if there is a reason to temporarily prefer a different default that is also ok).